user-guide/io.rst

```diff
@@ -6,5 +6,191 @@ I/O and file systems
+
+Single file, multiple writers without collective operations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+There are a number of ways to achieve this. For example, many processes can open the same file but access different parts by skipping some initial offset; parallel I/O libraries such as MPI-IO, HDF5 and NetCDF also enable this.
+
+Shared-file I/O has the advantage that all the data is organised correctly in a single file making analysis or restart more straightforward.
+
+The problem is that, with many clients all accessing the same file, there can be a lot of contention for file system resources.
+
+Single Shared File with collective writes (SSF)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The problem with having many clients performing I/O at the same time is that, to prevent them clashing with each other, the I/O library may have to take a conservative approach. For example, a file may be locked while each client is accessing it which means that I/O is effectively serialised and performance may be poor.
+
+However, if I/O is done collectively where the library knows that all clients are doing I/O at the same time, then reads and writes can be explicitly coordinated to avoid clashes. It is only through collective I/O that the full bandwidth of the file system can be realised while accessing a single file.
+
+
+Achieving efficient I/O
+-----------------------
+This section provides information on getting the best performance out of the parallel ``/work`` file systems on ARCHER2 when writing data, particularly using parallel I/O patterns.
+
+Lustre
+^^^^^^
+The ARCHER2 ``/work`` file systems use Lustre as a parallel file system technology. The Lustre file system provides POSIX semantics (changes on one node are immediately visible on other nodes) and can support very high data rates for appropriate I/O patterns.
+
+Striping
+^^^^^^^^
+One of the main factors leading to the high performance of Lustre file systems is the ability to stripe data across multiple Object Storage Targets (OSTs) in a round-robin fashion. Files are striped when the data is split up in chunks that will then be stored on different OSTs across the Lustre system. Striping might improve the I/O performance because it increases the available bandwith since multiple processes can read and write the same files simultaneously. However striping can also increase the overhead. Choosing the right striping configuration is key to obtain high performance results.
+
+Users have control of a number of striping settings on Lustre file systems. Although these parameters can be set on a per-file basis they are usually set on directory where your output files will be written so that all output files inherit the settings.
+
+
+Default configuration
+""""""""""""""""""""""
+ The ``/work`` file systems on ARCHER2 have the same default stripe settings:
+
+* A default stripe count of -1
```

Comment:
This is wrong. I think the default stripe count will be "1" (single stripe) rather than "-1" (maximal striping).